kuka_kr210_support/launch/load_kr210l150-2.launch

```diff
@@ -0,0 +1,3 @@
+<launch>
+  <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find kuka_kr210_support)/urdf/kr210l150-2.xacro'" />
```

Comment:
This line no loner works with most recent changes.

Reply:
This file is non-existent now.